libs/vkd3d/device.c

```diff
@@ -4514,6 +4514,14 @@ HRESULT d3d12_device_create(struct vkd3d_instance *instance,
         return E_INVALIDARG;
     }
 
+    if (!object->vk_info.KHR_timeline_semaphore)
+    {
+        ERR("Timeline semaphores are not supported by this implementation. This is required for correct operation.\n");
+        d3d12_device_destroy(object);
+        vkd3d_free(object);
+        return E_NOTIMPL;
+    }
+
```

Comment:
This feature check should probably be moved into `vkd3d_init_device_caps` and return `E_INVALIDARG` instead of `E_NOTIMPL`, to be consistent with the feature level check.

Reply:
Should be fixed now.